.github/ISSUE_TEMPLATE/release-qa-check-list.md

```diff
@@ -0,0 +1,142 @@
+---
+name: Release QA check list
+about: Check list for performing QA of a release candidate
+title: 'QA: check-list for vX.Y.Z (TT)'
+labels: testing
+assignees: ''
+
+---
+
+# App
+
+- [ ] verify that version numbers are reasonable
+- [ ] verify that settings are default
+
+# Websites test
+
+- [ ] verify that settings are default
+- [ ] run the test in foreground without specifying any option
+    - [ ] check that the time is decreasing
+- [ ] run the test in foreground and then put the app in background
+    - [ ] make sure that the time is making progress
+    - [ ] make sure that you receive a notification
+    - [ ] make sure that the test performed work in background
+- [ ] run the test with just https://ooni.torproject.org/ and https://expired.badssl.com/
+- [ ] run the test with only some categories
+    - [ ] make sure you did only actually test these categories
+- [ ] run the test with a very short test runtime
+- [ ] run the test without uploading measurements and then perform a background upload
```

Comment:
👍

Reply:
Maybe we can also add something about the counter of the upload decreasing?

Reply:
done!